frontend/src/metabase/collections/containers/CollectionContent.jsx

```diff
@@ -0,0 +1,588 @@
+import React from "react";
+import { Box, Flex } from "grid-styled";
+import _ from "underscore";
+import { t, msgid, ngettext } from "ttag";
+import cx from "classnames";
+import { connect } from "react-redux";
+import { withRouter } from "react-router";
+import { dissoc } from "icepick";
+
+import { entityTypeForObject } from "metabase/schema";
+
+import * as Urls from "metabase/lib/urls";
+import { color } from "metabase/lib/colors";
+
+import withToast from "metabase/hoc/Toast";
+import listSelect from "metabase/hoc/ListSelect";
+
+import Collection from "metabase/entities/collections";
+import Search from "metabase/entities/search";
+import EntityCopyModal from "metabase/entities/containers/EntityCopyModal";
+
+import CollectionMoveModal from "metabase/containers/CollectionMoveModal";
+
+import Button from "metabase/components/Button";
+import CreateDashboardModal from "metabase/components/CreateDashboardModal";
+import EntityMenu from "metabase/components/EntityMenu";
+import EntityItem from "metabase/components/EntityItem";
+import { Grid, GridItem } from "metabase/components/Grid";
+import Icon, { IconWrapper } from "metabase/components/Icon";
+import Link from "metabase/components/Link";
+import Modal from "metabase/components/Modal";
+import PageHeading from "metabase/components/PageHeading";
+import StackedCheckBox from "metabase/components/StackedCheckBox";
+import Tooltip from "metabase/components/Tooltip";
+import VirtualizedList from "metabase/components/VirtualizedList";
+import BulkActionBar from "metabase/components/BulkActionBar";
```

Comment:
I can't figure out how to use these in the UI. It looks like bulk actions weren't affected by this change, but wanted to call that out anyways.

Reply:
Yeah the cue to enter bulk actions is too subtle and we're going to rework that. Right now you hover on the icon of the item and that lets you click a checkbox and that puts the view into multi select mode. It's unchanged from the old version in this new branch.